src/sentry/tasks/integrations/github/language_parsers.py

```diff
@@ -96,4 +96,39 @@ def generate_function_name_conditions(function_names: List[str], stack_frame: in
         )
 
 
+class JavascriptParser(LanguageParser):
+    @staticmethod
+    def extract_functions_from_patch(patch: str) -> Set[str]:
+        r"""
+        Type of function declaration    Example
+        Function declaration:           function hello(argument1, argument2)
+        Arrow function:                 export const blue = (argument) => {
```

Comment:
The word export here doesn't really matter, right?

Reply:
it doesn't matter, but i included it as an example of some words in front of the `constant` that we also handle